### ISSUE TITLE
Errors printed

### ISSUE THREAD

**Issue by polyvertex** [AvatarHurden/keypirinha-currency#2]
Hi, with the v1.2 of the package, my Keypirinha console keep spitting the following:

```
** ERROR: Error from Currency.Currency.on_suggest: <class 'AttributeError'>: 'NoneType' object has no attribute 'validate_codes'
Traceback (most recent call last):
  File "D:\prog\apps\keypirinha\portable\Profile\InstalledPackages\Currency.keypirinha-package\currency.py", line 112, in on_suggest
  File "D:\prog\apps\keypirinha\portable\Profile\InstalledPackages\Currency.keypirinha-package\currency.py", line 205, in _parse_and_merge_input
```

**Comment by AvatarHurden** [comment]
This was an issue with the local rates file, and should be fixed in version 1.2.1.

If it happens again, please let me know and I'll reopen the issue.

**Comment by polyvertex** [comment]
Didn't occur so far since the update. Thanks!

**Comment by markcs** [comment]
Sorry to comment on a closed issued, but I'm seeing this same error with the latest update, but with different line numbers.

I'm not sure how to check the version, but it is the latest downloaded from github.  Any ideas?

**Comment by AvatarHurden** [comment]
Could you add the console log when the error occurs? If it's the same bug, I'll reopen the issue, otherwise it would be better to open a new one.

**Comment by markcs** [comment]
@AvatarHurden these are the errors:

09:38:37.804 ** ERROR: Error from Currency.Currency.on_suggest: <class 'AttributeError'>: 'NoneType' object has no attribute 'validate_codes'
Traceback (most recent call last):
  File "C:\Users\me\Programs\Keypirinha\portable\Profile\InstalledPackages\Currency.keypirinha-package\currency.py", line 105, in on_suggest
  File "C:\Users\me\Programs\Keypirinha\portable\Profile\InstalledPackages\Currency.keypirinha-package\currency.py", line 194, in _parse_and_merge_input

**Comment by AvatarHurden** [comment]
I'm not exactly sure what the issue is, but I added a new version fixing one bug that I found. Hopefully this fixes the problem.

Just out of curiosity, is there a "rates.json" file in your Local folder for the plugin and, if there is, could you attach it in another comment? I think the issues are coming from problems in this file, and another example could help me test this theory 😄

**Comment by markcs** [comment]
@AvatarHurden I do not have the rates.json file.  I guess that is the problem....

**Comment by polyvertex** [comment]
In case it helps, after some tests I observed the same symptoms after deleting rates.json.

Also please take care of removing any debug printing before shipping your package, I have what seems to be the content of rates.json dumped to my console

**Comment by AvatarHurden** [comment]
Sorry about the dumps, I attached the wrong build to the release. It should be fixed now.

Are the errors still appearing in the latest version?

**Comment by polyvertex** [comment]
Seems ok after the update here, thanks!

**Comment by AvatarHurden** [comment]
Seems like the API I am using is returning an invalid JSON. Should be fixed in the latest release (1.2.3).
